cpp/example_code/s3/README.md

```diff
@@ -9,9 +9,16 @@ Amazon S3 is an object storage service that offers industry-leading scalability,
 This example uses the S3Client, which is a fully-featured S3 interface and is ideally suited for smaller files.  For other classes provided by the AWS SDK for C++ that also interface to
 Amazon S3, see example folder [s3-crt](../s3-crt) and example folder [transfer-manager](../transfer-manager).
 
+## ⚠️ Important
+* Running this code might result in charges to your AWS account. 
+* Running the tests might result in charges to your AWS account.
+* We recommend that you grant your code least privilege. At most, grant only the minimum permissions required to perform the task. For more information, see [Grant least privilege](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#grant-least-privilege). 
+* This code is not tested in every AWS Region. For more information, see [AWS Regional Services](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services).
+
+
 ## Code examples
-This is a workspace where you can find AWS SDK for C++ S3 examples utilizing the S3Client.
 
+### Single actions
 - [Copying an object](./copy_object.cpp) (CopyObject)
 - [Creating an Amazon S3 bucket](./create_bucket.cpp) (CreateBucket)
```

Comment:
For this and the following mentions of "Amazon S3 bucket", we can use "S3 bucket".

Also, in the "Making requests to..." link name, remove the extra space between "to" and "Amazon S3".

Reply:
changed